src/Catalyst.Core.Lib/Util/KeyUtil.cs

```diff
@@ -32,7 +32,7 @@ public static class KeyUtil
     {
         public static string KeyToString(this byte[] keyAsBytes)
         {
-            return Base32.ToBase32(keyAsBytes).ToLowerInvariant();
+            return Base32.ToBase32(keyAsBytes).ToLowerInvariant(); ///@TODO get from hashlib
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)